src/CattyTests/Utils/HandleLoginWithResponseTest.swift

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
Since you're testing the LoginViewController I would prefer naming this test to "LoginViewControllerTests".